src/spectrecoff-cli/commands/Panel.fs

```diff
@@ -39,4 +39,42 @@ type PanelDocumentation() =
     
     override _.Execute(_context, _) = 
         Cli.Theme.setDocumentationStyle
+
+        NewLine |> toConsole
+        pumped "Panel module"
+        |> alignedRule Left
+        |> toConsole
+
+        Many [
+            CO [
+                C "This module provides functionality from the panel widget of Spectre.Console ("
+                Link "https://spectreconsole.net/widgets/panel"
+                C ")"
+            ]
+            NL
+            C "The panel can be used by the panel function:"
+            BI [P "panel: (header: string) -> (content: OutputPayload) -> Renderable"]
+            NL
+            CO [
+                C "While the content can be an arbitrary payload" 
+                E "(known bug: Many does not work inside panel, fix on the way)"
+                C "the header needs to be a string."
+                C "Spectre still will render markup here, though, so all"; 
+                P "stringifyable payloads"; 
+                C "can be used here when mapped to string using the extension"; 
+                P "payload.toMarkedUpString."
```

Comment:
A little suggestion: I would return to my calm voice for the punctuation:

<img width="507" alt="image" src="https://user-images.githubusercontent.com/17160319/205434388-c5bbbc6b-3e58-4853-8933-0d30b4b5b003.png">

Reply:
I'm aware of that drawback. But currently, we are inserting spaces between the payloads in collections, so if you calm the punctuation, it will be separated by a space from the previous word :/

Reply:
I'd leave it for this PR, let's think about the problem separately, though.